AI4PEX_Data_Questionnaire/questionnaire.md

```diff
@@ -0,0 +1,61 @@
+We'd appreciate it if you had 5min to take this survey, this will help us gauge the amount of compute
+resources we'd need to have for the AI4PEX project (both available, and to request new allocations). Very many thanks 
+in advance! :beer:
+
```

Comment:
I would add a question on ML methods if they already know - what ML methods and if Deep Learning will be used.
I would also add one final question as an open "Other" question where they can fill out anything else they want to communicate w.r.t data.

Reply:
very good point! Added in https://github.com/AI4PEX/Housekeeping/pull/3/commits/4337e19c59dba235e5dff5057ab9739a70f84f92

Reply:
needs to clean up the language :laughing: